src/Loggers/Loggers.cpp

```diff
@@ -182,9 +194,7 @@ void Loggers::buildLoggers(Poco::Util::AbstractConfiguration & config, Poco::Log
 
     bool should_log_to_console = isatty(STDIN_FILENO) || isatty(STDERR_FILENO);
     bool color_logs_by_default = isatty(STDERR_FILENO);
-
-    if (config.getBool("logger.console", false)
-        || (!config.hasProperty("logger.console") && !is_daemon && should_log_to_console))
+    if (config.getBool("logger.console", false) || (!config.hasProperty("logger.console") && !is_daemon && should_log_to_console))
```

Comment:
Here and in other places I think these `if`'s were easier to read before the changes. Some deleted empty lines I think also could be brought back :)

Reply:
Made changes in Loggers.cpp. Hope this will look nicer :)